src/mon/AuthMonitor.cc

```diff
@@ -1801,19 +1801,6 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
       + " tag " + pg_pool_t::APPLICATION_NAME_CEPHFS
       + " data=" + filesystem;
 
-    std::map<string, bufferlist> wanted_caps = {
-      { "mon", _encode_cap(mon_cap_string) },
-      { "osd", _encode_cap(osd_cap_string) },
-      { "mds", _encode_cap(mds_cap_string) }
-    };
-
-    if (!valid_caps("mon", mon_cap_string, &ss) ||
-        !valid_caps("osd", osd_cap_string, &ss) ||
-	!valid_caps("mds", mds_cap_string, &ss)) {
-      err = -EINVAL;
-      goto done;
-    }
-
     EntityAuth entity_auth;
     if (mon.key_server.get_auth(entity, entity_auth)) {
       for (const auto &sys_cap : wanted_caps) {
```

Comment:
Which `wanted_caps` do we use here after removing the `wanted_caps` definition several lines above?